### PR TITLE
Circumvent Go's GC

### PR DESCRIPTION
**Pull request by andreibancioiu**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,10 +1,16 @@
...
 	go build ./...
 
-test:
+build-arwen:
+	go build -o ./cmd/arwen/arwen ./cmd/arwen
+	cp ./cmd/arwen/arwen ./ipc/tests
+
+test: build-arwen run-tests
+
+run-tests:
```
Comment (camilbancioiu) [inline Makefile]: Add dependency on `clean` as well (i.e. `run-tests: clean`). I forgot to add it to `test`.

File: Makefile [hunk 2]
```diff
@@ -1,10 +1,16 @@
-.PHONY: test build clean
+.PHONY: test build build-arwen clean
```
Comment (camilbancioiu) [inline Makefile]: `run-tests` should be added here as well. All `make` targets are phony, unless they are real filenames.

Comment (andreibancioiu) [reply Makefile]: Added.

File: Makefile (hunk 1 above)
Comment (andreibancioiu) [reply Makefile]: Added.

**Pull request merged**